k8s/coordinator/development/config-release-coordinator.toml

```diff
@@ -1,21 +1,14 @@
-aggregator_url = "http://dev-aggregator.xain.io:80"
-
-[logging]
-filter = "info,xain_fl=debug"
-
 [api]
 bind_address = "0.0.0.0:8081"
 
-[rpc]
-bind_address = "0.0.0.0:5555"
-aggregator_address = "aggregator-service:6666"
-
-[federated_learning]
-rounds = 100
-participants_ratio = 0.8
-min_clients = 100
-heartbeat_timeout = 5
-
-[metric_store]
-database_url = "http://influxdb-development:8086"
-database_name = "metrics"
+[pet]
```

Comment:
Maybe the file name can be updated to reflect the new values? perhaps something like "config.toml" (it is under `coordinator/` after all)

Reply:
Good point!